src/plugins/rolldice-plugin.vala

```diff
@@ -0,0 +1,164 @@
+/*
+ * Copyright (C) 2017 André Nasturas <andre.nasturas+synapse@delfosia.net>
+ *
+ * This program is free software; you can redistribute it and/or modify
+ * it under the terms of the GNU General Public License as published by
+ * the Free Software Foundation; either version 2 of the License, or
+ * (at your option) any later version.
+ *
+ * This program is distributed in the hope that it will be useful,
+ * but WITHOUT ANY WARRANTY; without even the implied warranty of
+ * MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the
+ * GNU General Public License for more details.
+ *
+ * You should have received a copy of the GNU General Public License
+ * along with this program; if not, write to the Free Software
+ * Foundation, Inc., 51 Franklin St, Fifth Floor, Boston, MA 02110-1301  USA. *
+ */
+
+namespace Synapse
+{
+  public class RolldicePlugin : Object, Activatable, ItemProvider
+  {
+    public bool enabled { get; set; default = true; }
+
+    public void activate ()
+    {
+
+    }
+
+    public void deactivate ()
+    {
+
+    }
+
+    private class Result : TextMatch
```

Comment:
could this be whats causing the "Send to" action? Have you tried using `ActionMatch`?

Reply:
If I do, I have to implement the ```do_action()``` method with a specific action, and I don't know what action could be relevant in this case.
Since the plugin returns only a value, the default actions are relevant, only the order could be improved (the "copy to pastebin" before "send to" for example).